webapp/javascript/redux/reducers/continuous.ts

```diff
@@ -379,7 +379,7 @@ export const continuousSlice = createSlice({
       state.tagExplorerView = {
         ...state.tagExplorerView,
         activeTagProfile: action.payload.profile,
-        type: 'loaded',
+        type: state.tagExplorerView.type,
```

Comment:
Changed this line because there is kind of race condition for tag explorer loading state. Because `fetchTagExplorerViewProfile` and `fetchTagExplorerView` have access to `state.tagExplorerView.type` at the same time. They fetch different data but share loading state between each other. It causes false `type: loaded` sometimes. This update isn't the best but working solution. Any thoughts? Ideally we need to separate loading state for them.

Reply:
I would separate it in different states, yes.

Reply:
See the labels loading for more or less how I would implement this. Each part has its own loading state.